plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1709,9 +1710,13 @@ elementgui.common.event_on_crafted=When item is crafted/smelted
 elementgui.common.event_on_block_placed_by=When block is placed by
 elementgui.common.event_on_random_update=Client display random tick
 elementgui.common.event_on_block_hit_by_projectile=On block hit by projectile
+elementgui.common.event_is_bonemeal_target=Is valid bone meal target
```

Comment:
Maybe something like "Does bone meal work on this block" would be more user-friendly? Or something similar in these words